docker/examples/docker-compose-kafka.yml

```diff
@@ -29,10 +29,7 @@ services:
     # environment:
       # - KAFKA_ADVERTISED_HOST_NAME=192.168.99.100
     ports:
-      - 2181:2181
```

Comment:
we don't need to bind host ports by default as it is especially confusing as only 19092 will work
the others are exposed directly by the Dockerfile so we don't need to re-declare expose here either.

Reply:
👍